torch/_inductor/lowering.py

```diff
@@ -297,6 +297,26 @@ def wrapped(*args, **kwargs):
         return out
 
     aten_fn = get_overloads(aten_fn)
+    core_aten_ops = core_aten_decompositions()
+
+    for fn in aten_fn:
+        if not isinstance(fn, torch._ops.OpOverload):
```

Comment:
When is it not an OpOverload?

Reply:
It can be a HigherOrderOperator, for example, `run_with_rng_state`

Reply:
Can we change this to be `if isinstance(fn, HigherOrderOperator): ... assert(fn, torch._ops.OpOverlaod)` to be a bit more explicit of what's going on?